### DISCUSSION TITLE
reduced dynamic orbits ，LEO

### DISCUSSION THREAD

**Discussion by LNTU-YS** [groops-devs/groops#203] [Q&A] [answered]
Hello,

We are currently conducting reduced dynamic orbit calculations for GRACE-FO but we encountered some problems. Imitating the example [GNSS satellite orbit determination and station network analysis] you provided, I have created a reduced dynamic orbit script for GRACE-FO. When I run the script, it will report an error, as shown in the following figure.
![image](https://github.com/groops-devs/groops/assets/111558200/f31e43c6-2829-4a7c-a264-e9f6a919a0f3)
![image](https://github.com/groops-devs/groops/assets/111558200/a2fa0b43-85b2-479a-9ff8-0ae9bcafedae)
I think the problem may be in setting the time. According to my understanding, the error feedback from the script is that if I want to calculate the orbit for a day, I need more data to calculate than for a day. But when I used one day's data to calculate a 20 hour orbit (shortened by two hours before and after each), the error reported was the same. Of course, this is just my immature idea. I'm not sure if the problem really arises here.

Thank you in advance for your feedback and help.
With Kind Regards
YUAN

**Comment by tez1994** [comment]
Hello，Yuan
It seems like you select only 1 step in 10 to estimate the parameters, you should use all 8640 epochs.
Now there is already example about grace-fo RD POE on ftp site of groops https://ftp.tugraz.at/outgoing/ITSG/groops/
you may start with that to get a better understanding of procedure of  RD POE.

**Reply by LNTU-YS** [reply]
Hello,

Thank you very much for your reminder. I really didn't notice the example of [scenario Leo Reduced Dynamic Orbit] before. I will study it carefully and hope to solve my problem.
Regarding the "It seems like you select only 1 step in 10 to estimate the parameters, you should use all 8640 epochs." you mentioned, the problem is not here, because I have also tried this and still have a similar problem.

With Kind Regards
YUAN

**Comment by tmayerguerr** [comment, marked as answer]
Hello Yuan,

You only have measurements for one GRACE satellite. However, it seems that you are trying to determine many more parameters than is possible. For the determination of Earth rotation parameters you need ground stations. For the determination of GPS satellite clock errors you need a large number of receivers. You should only set parameters for the orbit determination of the GRACE satellites.

regards
Torsten
